iis-workflows/iis-workflows-top/src/main/resources/eu/dnetlib/iis/workflows/top/preprocessing/main/oozie_app/workflow.xml

```diff
@@ -119,6 +119,12 @@
 			<description>metadata extraction cache location, path pointing to
 				root cache directory holding meta.json file</description>
 		</property>
+		<property>
+            <name>referenceextraction_project_fundingclass_blacklist_regex</name>
```

Comment:
For a moment there I was wondering if you should use short of "regex" or "regexp." People on [SO](http://stackoverflow.com/questions/3058246/which-is-the-correct-shorthand-regex-or-regexp) say that both are Ok, don't miss the [first answer](http://stackoverflow.com/a/3058259/817499).